api/v1alpha1/jobset_types.go

```diff
@@ -86,6 +102,38 @@ type Network struct {
 	EnableDNSHostnames *bool `json:"enableDNSHostnames,omitempty"`
 }
 
+type TerminationPolicyTarget string
+
+const (
+	TerminationPolicyTargetAll TerminationPolicyTarget = "All"
+	TerminationPolicyTargetAny TerminationPolicyTarget = "Any"
```

Comment:
add comment:

// Any means any of the jobs managed by the jobset.

Note that AnyOf is the operator that will allow selecting any of a subset of jobs.